udata/api/oauth2.py

```diff
@@ -70,7 +72,7 @@ class OAuth2Client(ClientMixin, db.Datetimed, db.Document):
                           thumbnails=[150, 25])
 
     redirect_uris = db.ListField(db.StringField())
-    scopes = db.ListField(db.StringField(), default=['default'])
+    scope = db.StringField(default='')
```

Comment:
Any particular reason for this change (aside that we don't use any scopes)?

Reply:
It's basically a copy of the way this is handled in authlib doc